From e3189bb82c8e7a2bca59ed1dd5efb560944f7a1c Mon Sep 17 00:00:00 2001
From: David Cohen <david.a.cohen@intel.com>
Date: Tue, 18 Jun 2013 16:10:13 -0700
Subject: [PATCH 012/429] intel-mid, sdhci: fixing warning in
 init_sdhci_get_data()

init_sdhci_get_data() should return int value, but currently returns no
value. This patch fixes it.

Signed-off-by: David Cohen <david.a.cohen@intel.com>
---
 .../x86/platform/intel-mid/device_libs/pci/platform_sdhci_pci.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/x86/platform/intel-mid/device_libs/pci/platform_sdhci_pci.c b/arch/x86/platform/intel-mid/device_libs/pci/platform_sdhci_pci.c
index 92f46dbc2c1c..e715a2d4b153 100644
--- a/arch/x86/platform/intel-mid/device_libs/pci/platform_sdhci_pci.c
+++ b/arch/x86/platform/intel-mid/device_libs/pci/platform_sdhci_pci.c
@@ -139,6 +139,8 @@ struct sdhci_pci_data *mmc_sdhci_pci_get_data(struct pci_dev *pci_dev, int slotn
 static int __init init_sdhci_get_data(void)
 {
 	sdhci_pci_get_data = mmc_sdhci_pci_get_data;
+
+	return 0;
 }
 
 arch_initcall(init_sdhci_get_data);
-- 
2.37.3

